languages/elixir/config.json

```
@@ -33,6 +33,12 @@
         "concepts": ["lists", "string-literals"],
         "prerequisites": ["booleans"]
       },
+      {
+        "slug": "anonymous-functions",
+        "uuid": "1e3ceb20-715c-4157-9192-13284217affc",
+        "concepts": ["anonymous-functions", "closures", "bitwise"],
```

Maybe for consistency with the C# track's naming (unless this makes no sense for Elixir):

```suggestion
        "concepts": ["anonymous-functions", "closures", "bit-manipulation"],
```